### DISCUSSION TITLE
SSR not working?

### DISCUSSION THREAD

**Discussion by mkonars** [vercel/next.js#54044] [Help] [answered]
### Summary

Hi,
Here is an example of simple SSR caching https://stackblitz.com/edit/github-nnzyup
However, I see a new value on each refresh. Could anyone tell me why is that?

**Comment by gerardnico** [comment]
`cache-control` manage the cache of HTTP response.

The cache is the cache of the application in the HTTP chain (ie browser or proxy).
https://datacadamia.com/web/http/cache/cache#chain

It means that the Stackblitz browser (the right panel) doesn't have any cache?

It's easier to check with a browser against a local service.

**Reply by mkonars** [reply]
Hi, my understanding was that the framework is doing some magic to cache the response on the server, so each client would get the same response. I don't want to depend on the response headers.

I have just started learning about next.js, but what I would like to achieve is to make one db call in the component to fetch initial data and then have a static page with the same response for all the requests. 
Something like this. I would expect this Page to be rendered just once or the fetch to be executed once, but none happens. 
Do you know why? I am using next.js v13 with the new app router.

```
import { cache } from "react"

export const revalidate = 1000

export const getItem = cache(async () => {
  const res = await fetch("http://localhost:5003/ping", {
    headers: {
      "Content-Type": "application/json",
    },
  })
  const text = await res.text()

  return text
})

export default async function Page() {
  const json = await getItem()

  return (
    <main className="flex min-h-screen flex-col items-center justify-between p-24">
      <div>{json}</div>
    </main>
  )
}
```

**Comment by mkonars** [comment, marked as answer]
Ok, I figured it out (or at least some part of it).
The main issue was that I had dev tools opened in chrome and it was sending the no cache headers. Found it by reading this https://github.com/vercel/next.js/issues/53567

I also tried to cache something like "new Date", but I assume next.js detects this somehow as dynamic function. Don't know how though, cause it's not mentioned anywhere. The solution I found for this is to create local route handler and fetch it. Cache will work in this case.

**Reply by mkonars** [reply]
Also realised that SSR caching does not work in DEV mode. It will add no-store and must-revalidate headers to the response.
